integrationTests/cypress/support/step_definitions/then.js

```diff
@@ -0,0 +1,104 @@
+const {
+    Then
+} = require("cypress-cucumber-preprocessor/steps");
+
+const env = require('@cloudogu/dogu-integration-test-library/lib/environment_variables')
+
+Then(/^the page shows the replicated data of the user in tabular form$/, function () {
+    cy.fixture("testuser_data").then((testuserdata) => {
+        cy.get("[id=login]").then(loginFromSite => {
```

Comment:
I think you can use `contains` https://docs.cypress.io/api/commands/contains#Syntax:
```suggestion
        cy.get("[id=login]").contains(testuserdata.username)
```